src/reducers/entities/posts.ts

```diff
@@ -1052,7 +1052,7 @@ export function reactions(state = {}, action) {
 }
 
 function storeReactionsForPost(state, post) {
-    if (!post.metadata || !post.metadata.reactions) {
+    if (!post.metadata || !post.metadata.reactions || post.delete_at > 0) {
         return state;
```

Comment:
what happens if the post already has reactions, isn't this `state` already including them?

Reply:
This is called when the channel is refreshed and `RECEIVED_POSTS/RECEIVED_POST` is triggered so the deleted message was previously removed (and corresponding reactions) when the channel was changed and is coming back in again with the reactions that should not be added. 

When you are in the channel and the message is deleted, the reaction is correctly removed at the time from the state.

Reply:
It's strange that the server is sending this information for deleted posts. I'm going to file a ticket to look into metadata being provided for deleted posts